ClientApp/app/containers/not-found/not-found.component.ts

```diff
@@ -3,7 +3,6 @@ import { Component, OnInit } from '@angular/core';
 @Component({
     selector: 'not-found',
     templateUrl: './not-found.component.html'
-    // styleUrls: ['./not-found.component.css']
```

Comment:
This line caused the angular 2 template loader to fail requiring the not-found.component.html